plugins/woocommerce/includes/admin/helper/class-wc-helper-admin.php

```diff
@@ -58,28 +58,28 @@ public static function add_marketplace_settings( $settings ) {
 	 * @return string
 	 */
 	public static function get_connection_url() {
+		global $current_screen;
+
+		$connect_url_args = array(
+			'page'                 => 'wc-addons',
+			'section'              => 'helper',
+		);
```

Comment:
I'd fix this formatting thing. Not a blocker but looks nicer.

```suggestion
		$connect_url_args = array(
			'page'    => 'wc-addons',
			'section' => 'helper',
		);
```